
From: Arnd Bergmann <arnd@arndb.de>

cpumask-5-10-rewrite-cpumaskh-single-bitmap-based from 2.6.7-rc3-mm1
causes include2/asm/smp.h:54:1: warning: "cpu_online" redefined

Signed-off-by: Arnd Bergmann <arnd@arndb.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-s390/smp.h |    6 ------
 1 files changed, 6 deletions(-)

diff -puN include/asm-s390/smp.h~s390-fix-cpu_online-redefined-warnings include/asm-s390/smp.h
--- 25/include/asm-s390/smp.h~s390-fix-cpu_online-redefined-warnings	Fri Jun 11 15:09:24 2004
+++ 25-akpm/include/asm-s390/smp.h	Fri Jun 11 15:09:24 2004
@@ -31,10 +31,6 @@ typedef struct
 
 extern int smp_call_function_on(void (*func) (void *info), void *info,
 				int nonatomic, int wait, int cpu);
-
-extern cpumask_t cpu_online_map;
-extern cpumask_t cpu_possible_map;
-
 #define NO_PROC_ID		0xFF		/* No processor magic marker */
 
 /*
@@ -51,8 +47,6 @@ extern cpumask_t cpu_possible_map;
 
 #define smp_processor_id() (S390_lowcore.cpu_data.cpu_nr)
 
-#define cpu_online(cpu) cpu_isset(cpu, cpu_online_map)
-
 extern __inline__ __u16 hard_smp_processor_id(void)
 {
         __u16 cpu_address;
_
